schema/model.py

```diff
@@ -139,6 +143,9 @@ class Project(StrictBaseModel):
     pypi_options: PyPIOptions | None = Field(
         None, alias="pypi-options", description="Options related to PyPI indexes for this project"
     )
+    preview: list[KnownPreviewFeature | str] | bool | None = Field(
```

Comment:
Instead of a `class` I think it makes more sense to use `Literal["new_solve"]`

Reply:
But I can't make an empty version for the mean time right?